src/bot/controllers/DetailsController.js

```diff
@@ -0,0 +1,52 @@
+const BaseController = require('./BaseController');
+
+class DetailsController extends BaseController {
+    detailsImage(venue) {
```

Comment:
better move this methods to separate helpers, because they are independent from controller data